src/SignalR/src/Microsoft.AspNetCore.SignalR.Common/Protocol/HubMethodInvocationMessage.cs

```diff
@@ -112,7 +112,7 @@ public override string ToString()
 
             try
             {
-                streamIds = string.Join(", ", StreamIds?.Select(id => id?.ToString()));
+                streamIds = string.Join(", ", StreamIds != null ? StreamIds.Select(id => id?.ToString()) : Array.Empty<string>());
```

Comment:
Should an error be thrown for a null streamId?

This place might not be the right place but I think it should be guarded against.

Reply:
I don't think we should care if stream ids is null, it's similar to how we don't care about extra data in the protocol